### PR TITLE
feat(natives/object): add documentation for DELETE_OBJECT

### PR DESCRIPTION
**Pull request by ahcenezdh**
* Add documentation for DELETE_OBJECT
* Add examples (Lua, JS, C#)
* A note about some entities who can't be deleted

### CODE REVIEW

File: OBJECT/DeleteObject.md [hunk 1]
```diff
@@ -1,17 +1,71 @@
...
+```
+
+```csharp
+using static CitizenFX.Core.Native.API;
+
+int playerPed = PlayerPedId();
+Vector3 playerCoords = GetEntityCoords(playerPed, false);
+uint objectHash = GetHashKey("v_ret_gc_chair03");
+
+int object = GetClosestObjectOfType(playerCoords.X, playerCoords.Y, playerCoords.Z, 10.0f, objectHash, true, false, false);
```
Comment (AvarianKnight) [inline OBJECT/DeleteObject.md:62]:
```suggestion
int prop = GetClosestObjectOfType(playerCoords.X, playerCoords.Y, playerCoords.Z, 10.0f, objectHash, true, false, false);
```

File: OBJECT/DeleteObject.md [hunk 2]
```diff
@@ -1,17 +1,71 @@
...
+
+// If the object is a mission entity, we have to set it to false. Otherwise, it won't be possible to delete the object.
+if (IsEntityAMissionEntity(object))
```
Comment (AvarianKnight) [inline OBJECT/DeleteObject.md:65]:
```suggestion
if (IsEntityAMissionEntity(prop))
```

File: OBJECT/DeleteObject.md [hunk 3]
```diff
@@ -1,17 +1,71 @@
...
+{
+    SetEntityAsMissionEntity(object, false, true);
```
Comment (AvarianKnight) [inline OBJECT/DeleteObject.md:67]:
```suggestion
    SetEntityAsMissionEntity(prop, false, true);
```

**Review by AvarianKnight — CHANGES_REQUESTED** [review]
`object` is a reserved word in c#

**Comment by ahcenezdh** [comment]
Those exemples were tested (Lua, JS) , by deleting object in a IPL (the chair) with SetEntityAsMissionEntity

File: OBJECT/DeleteObject.md [hunk 4]
```diff
@@ -1,17 +1,71 @@
...
+
+DeleteObject(object);
+```
+
+```csharp
+using static CitizenFX.Core.Native.API;
+
+int playerPed = PlayerPedId();
+Vector3 playerCoords = GetEntityCoords(playerPed, false);
+uint objectHash = GetHashKey("v_ret_gc_chair03");
```
Comment (AvarianKnight) [inline OBJECT/DeleteObject.md:60]:
```suggestion
uint objectHash = Game.GenerateHashASCII("v_ret_gc_chair03");
```

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

**Pull request merged**